app/controllers/records_controller.rb

```diff
@@ -42,16 +47,17 @@ class RecordsController < ApplicationController
 
   # edit
   get '/records/:id/edit' do
-    @record = Records.find(params[:id])
-    p "Record is retrieved: id = #{params[:id]}"
-    p "User id for requested record: #{current_user.id}"
-
-    check_access_to_records(@record, :'records/edit')
+    id = params[:id]
+    error_if_not_logged_in
+    @record = find_record(id)
+    erb :'/records/edit'
   end
 
   patch '/records/:id/edit' do
-    redirect_if_not_logged_in
-    @record = Records.find(params[:id])
+    id = params[:id]
+    error_if_not_logged_in
+    @record = find_record(id)
```

Comment:
if id using only in this row could be, it's ok if pameter definition is short
@record = find_record(params[:id])